src/main/webapp/app/quiz/edit/drag-and-drop-question/edit-drag-and-drop-question.component.ts

```diff
@@ -1,30 +1,43 @@
-import { AfterViewInit, Component, ElementRef, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChanges, ViewChild } from '@angular/core';
-import { DragAndDropQuestion } from '../../../entities/drag-and-drop-question';
-import { ArtemisMarkdown } from '../../../components/util/markdown.service';
-import { DragAndDropQuestionUtil } from '../../../components/util/drag-and-drop-question-util.service';
-import { FileUploaderService } from '../../../shared/http/file-uploader.service';
-import { DropLocation } from '../../../entities/drop-location';
-import { DragItem } from '../../../entities/drag-item';
-import { DragAndDropMapping } from '../../../entities/drag-and-drop-mapping';
-import { DragAndDropMouseEvent } from '../../../entities/drag-item/drag-and-drop-mouse-event.class';
-import { DragState } from '../../../entities/drag-item/drag-state.enum';
-import { AceEditorComponent } from 'ng2-ace-editor';
+import {
+    Component,
+    ElementRef,
+    EventEmitter,
+    Input,
+    OnChanges,
+    OnInit,
+    Output,
+    SimpleChanges,
+    ViewChild,
+    ChangeDetectorRef
+} from '@angular/core';
+import { DragAndDropQuestion } from 'app/entities/drag-and-drop-question';
+import { ArtemisMarkdown } from 'app/components/util/markdown.service';
+import { DragAndDropQuestionUtil } from 'app/components/util/drag-and-drop-question-util.service';
+import { FileUploaderService } from 'app/shared/http/file-uploader.service';
+import { DropLocation } from 'app/entities/drop-location';
+import { DragItem } from 'app/entities/drag-item';
+import { DragAndDropMapping } from 'app/entities/drag-and-drop-mapping';
+import { DragAndDropMouseEvent } from 'app/entities/drag-item/drag-and-drop-mouse-event.class';
+import { DragState } from 'app/entities/drag-item/drag-state.enum';
 import * as $ from 'jquery';
-import 'brace/theme/chrome';
-import 'brace/mode/markdown';
 import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
 import * as TempID from 'app/quiz/edit/temp-id';
+import { HintCommand } from 'app/markdown-editor/specialCommands/hint.command';
```

Comment:
can we export commands in one index.ts file and import them all together from the same path?